lib/storageBuilder.js

```diff
@@ -15,6 +15,10 @@ var AuthSdkError = require('./errors/AuthSdkError');
 
 // storage must have getItem and setItem
 function storageBuilder(webstorage, storageName) {
+  if (typeof storageName !== 'string' || !storageName.length) {
```

Comment:
Curious how this check came about. Was `getStorage()` returning `undefined`?

Reply:
I had defined the PKCE storage with a key constant, but never defined the constant. turns out everyone was fine with an undefined storageName.